blender/arm/material/cycles_nodes/nodes_shader.py

```diff
@@ -34,12 +34,12 @@ def parse_mixshader(node: bpy.types.ShaderNodeMixShader, out_socket: NodeSocket,
     state.curshader.write('{0}float {1} = clamp({2}, 0.0, 1.0);'.format(prefix, fac_var, fac))
     state.curshader.write('{0}float {1} = 1.0 - {2};'.format(prefix, fac_inv_var, fac_var))
 
+    bc1, rough1, met1, occ1, spec1, opac1, rior1, emi1 = c.parse_shader_input(node.inputs[1])
     mat_state.emission_type = mat_state.EmissionType.NO_EMISSION
-    bc1, rough1, met1, occ1, spec1, opac1, emi1 = c.parse_shader_input(node.inputs[1])
```

Comment:
This is still wrong (also a few lines below and in `parse_addshader`), the shader input parsing needs to happen _after_ the emission type is temporarily set to `NO_EMISSION`.

Reply:
Fixed that, it caused a smearing of the emissive light.